src/features/classification/security-controls/utils.js

```diff
@@ -15,47 +15,52 @@ import {
     SHARED_LINK_ACCESS_LEVEL,
 } from '../constants';
 
-const { SHARED_LINK, DOWNLOAD, EXTERNAL_COLLAB, APP } = ACCESS_POLICY_RESTRICTION;
+const { SHARED_LINK, DOWNLOAD, EXTERNAL_COLLAB, APP, WATERMARK } = ACCESS_POLICY_RESTRICTION;
 const { DEFAULT, WITH_APP_LIST, WITH_OVERFLOWN_APP_LIST } = APP_RESTRICTION_MESSAGE_TYPE;
 const { DESKTOP, MOBILE, WEB } = DOWNLOAD_CONTROL;
 const { BLOCK, WHITELIST, BLACKLIST } = LIST_ACCESS_LEVEL;
 const { COLLAB_ONLY, COLLAB_AND_COMPANY_ONLY, PUBLIC } = SHARED_LINK_ACCESS_LEVEL;
 
-const getShortSecurityControlsMessage = (controls: Controls): ?MessageItem => {
-    const { sharedLink, download, externalCollab, app } = controls;
+const getShortSecurityControlsMessage = (controls: Controls): Array<MessageItem> => {
+    const items = [];
+    const { sharedLink, download, externalCollab, app, watermark } = controls;
+
     // Shared link and external collab restrictions are grouped
     // together as generic "sharing" restrictions
     const sharing = (sharedLink && sharedLink.accessLevel !== PUBLIC) || externalCollab;
 
-    if (sharing && download && app) {
-        return { message: messages.shortAllRestrictions };
-    }
-
-    if (sharing && download) {
-        return { message: messages.shortSharingDownload };
-    }
-
-    if (sharing && app) {
-        return { message: messages.shortSharingApp };
-    }
-
-    if (download && app) {
-        return { message: messages.shortDownloadApp };
-    }
-
-    if (sharing) {
-        return { message: messages.shortSharing };
-    }
-
-    if (download) {
-        return { message: messages.shortDownload };
+    switch (true) {
```

Comment:
I feel like this was a bit simpler before. I think ideally the `switch` would be used for enums and each `case` would match a unique value, but here all cases resolve to `true`. I think the same logic should be possible with the existing `if`s.

Reply:
I was thinking about this too, using existing `if`s we would now gotta concat each with `else if`s since we don't return immediately, looks a bit bulky. Let me know how you think ?

Reply:
I think the `else if` would be easier to understand and would result in less lines of code. A `switch` of boolean values where all the cases resolve to `true` is not a common usage and feels like it could be error prone.